addStyles.js

```diff
@@ -171,12 +171,17 @@ function attachTagAttrs(element, attrs) {
 function addStyle(obj, options) {
 	var styleElement, update, remove;
 
+	if (obj.sourceMap && typeof options.sourceRoot === "string") {
```

Comment:
@rockmacaca This is not mandatory, more of a triage 😛 , but would it be a breaking change, if we change the signature of `sourceMap`  to `sourceMap: Boolean || { sourceRoot: '...' }` instead of adding a new option `sourceRoot` ? Shouldn't be, but needs to be verified